app/dot/cpp/repository/repository/BaseRepository.java

```diff
@@ -118,9 +158,28 @@ public T getFirstSorted(Sort sort) {
     }
   }
 
+  public void saveWithHistory(T entity) {
+    final var currentEntity = findById(entity.getId());
+
+    entity.setModifiedAt(Instant.now().getEpochSecond());
+
+    if (currentEntity == null) {
+      // entity creation
```

Comment:
do we really need these comments?

Reply:
I find this flow easier to understand with a comment. I wouldn't create a separate method called setEntityCreationFields just to get rid of a comment when the method will never be reused.